src/app.service.ts

```diff
@@ -141,19 +142,21 @@ export class AppService {
           'username, password, gender, birthdate, email all in string format required',
       });
     }
-    const user = await this.prismaService.user.findUnique({
-      where: {
-        username: username,
-        email: email,
-      },
-    });
+    // const user = await this.prismaService.user.findUnique({
+    //   where: {
+    //     username: username,
+    //     email: email,
+    //   },
+    // });
+    const user = null;
     if (user) {
       throw new ConflictException({
         error: 'Duplicate entry',
```

Comment:
Can we use Non Spaced error codes? Like DUPLICATE_ERROR. So we wont have to manage case sensitive and we can create enums for error codes